web/satellite/src/components/accessGrants/EmptyState.vue

```diff
@@ -0,0 +1,71 @@
+
+// Copyright (C) 2020 Storj Labs, Inc.
+// See LICENSE for copying information.
+<template>
+	<div class="empty-state">
+		<div class="empty-state__modal">
+			<Key />
+			<h4 class="empty-state__modal__heading">Create Your First Access Grant</h4>
+			<p class="empty-state__modal__subheading">Get started by creating an Access to interact with your Buckets</p>
+			<VButton
+				label="Create Access Grant +"
+				width="199px"
+				height="44px"
+				v-bind:style="{ margin: `0 auto`, marginTop: `25px`}"
```

Comment:
could be simplified like this `:style="{ margin: `0 auto`, marginTop: `25px`}"`

I mean, it's not necessary to use v-bind

and by the way, I would not use inline styles if it's not necessary. You could give a class name to a component and style it as a regular tag inside style tags